### ISSUE TITLE
Add read-timeout to dex:get

### ISSUE THREAD

**Issue by svetlyak40wt** [fukamachi/dexador#28]
Right now it is possible to DOS a service which uses dexador to access external resources, if these resources are extremely slow or just accept TCP connection but dont respond.

**Comment by svetlyak40wt** [comment]
Sir Fukamachi, that do you think about this issue. Should this be considered a problem? Or may be there is another solution?

**Comment by svetlyak40wt** [comment]
@fukamachi san, maybe this abovementioned workaround should be added to the README?

**Comment by fukamachi** [comment]
Sorry for the late reply.
I wish Dexador could have `:connect-timeout` and `:read-timeout` options, however, it's the only workaround for now.

**Comment by hjudt** [comment]
Yes, this would be a really good enhancement. My application uses dex without connection pool to query servers (see if they are online, monitor resources). While I provide a :timeout to dex:get, there still seems to be some other timeout that delays the operation until it fails. Using trivial-timeout or some sb deadline kills it in time but this means that connections/ports stay open, using up resources over time without freeing them.
